pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>org.hyperledger.fabric</groupId>
     <artifactId>fabric-gateway-java</artifactId>
-    <version>2.2.0</version>
+    <version>2.2.2</version>
```

Comment:
We are not publishing from the `main` branch so the version here doesn't need to be changed, although there's no harm in doing that. It's just unnecessary noise in the commit history